tests/coreml/test_cml_SupportVectorClassifierConverter.py

```diff
@@ -47,7 +47,7 @@ def validate_zipmap(self, model):
         self.assertTrue('classProbability' in node.output)
 
     def test_support_vector_classifier_binary_no_prob(self):
-        svm = self._fit_binary_classification(SVC())
+        svm = self._fit_binary_classification(SVC(gamma=0.5))
```

Comment:
Thanks for the fixing.
Should we add a partial function to wrap SVC(gamma=0.5) and add some comments about why gamma is 0.5?

Reply:
The default value changed in 0.20, 'auto' was deprecated. The current default value is 'auto_deprecated' and will be replaced by 'scale' later on. This should be revisited when sklearn 0.22 goes out. 0.20 introduced a couple of changes with default values.